eng/linkerTests/linkerCorrectnessTests.targets

```diff
@@ -0,0 +1,110 @@
+<Project>
+  <PropertyGroup>
+    <LinkerTestDir>$(ArtifactsBinDir)linkerTests/</LinkerTestDir>
```

Comment:
Nit: use `$([MSBuild]::NormalizeDirectory())` for dirs and `$($([MSBuild]::NormalizePath())` for full paths.

Reply:
Will fix.